src/basedtyping/transformer.py

```diff
@@ -0,0 +1,253 @@
+"""utilities to create standard compatible annotations"""
+
+from __future__ import annotations
+
+import ast
```

Comment:
I heard a rumor that the cool kids are using `libcst` nowadays

Reply:
yeah, will have to look into something other than ast, esp when i want non-standard syntax

Reply:
Let me know if you find anything like that; I'll probably be needing something like that in the future for `unpy`

Reply:
it's likely gonna involve forking rust and it's crates